gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/ExtractJobConfiguration.java

```diff
@@ -200,7 +199,7 @@ public Builder toBuilder() {
   }
 
   @Override
-  ToStringHelper toStringHelper() {
+  protected ToStringHelper toStringHelper() {
```

Comment:
We could probably leave this as package scope since `JobConfiguration` is now an abstract class, right?